tables/rx_potency/chen21e-pot.csv

```diff
@@ -0,0 +1,118 @@
+Chen21e,REGN10933,WA1+S:501Y+614G,Figure 1c,Virus isolate,IC50,12,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933,Chen21e-B.1.1.7,Figure 1c,Virus isolate,IC50,8,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933,Chen21e-B.1.351,Figure 1c,Virus isolate,IC50,9462,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933,Chen21e-B.1.1.28,Figure 1c,Virus isolate,IC50,5002,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933,Chen21e-B.1.429,Figure 1c,Virus isolate,IC50,4,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933,Chen21e-B.1.617.1,Figure 1c,Virus isolate,IC50,133,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933,Chen21e-B.1.526/S477N,Figure 1c,Virus isolate,IC50,4,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933,Chen21e-B.1.526/E4B4K,Figure 1c,Virus isolate,IC50,82,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10987,WA1+S:614G,Figure 1c,Virus isolate,IC50,12,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10987,WA1+S:501Y+614G,Figure 1c,Virus isolate,IC50,17,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10987,Chen21e-B.1.1.7,Figure 1c,Virus isolate,IC50,7,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10987,Chen21e-B.1.351,Figure 1c,Virus isolate,IC50,4,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10987,Chen21e-B.1.1.28,Figure 1c,Virus isolate,IC50,7,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10987,Chen21e-B.1.429,Figure 1c,Virus isolate,IC50,108,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10987,Chen21e-B.1.617.1,Figure 1c,Virus isolate,IC50,93,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10987,Chen21e-B.1.526/S477N,Figure 1c,Virus isolate,IC50,71,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10987,Chen21e-B.1.526/E4B4K,Figure 1c,Virus isolate,IC50,104,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933 + REGN10987,WA1+S:614G,Figure 1c,Virus isolate,IC50,5,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933 + REGN10987,WA1+S:501Y+614G,Figure 1c,Virus isolate,IC50,8,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933 + REGN10987,Chen21e-B.1.1.7,Figure 1c,Virus isolate,IC50,3,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933 + REGN10987,Chen21e-B.1.351,Figure 1c,Virus isolate,IC50,8,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933 + REGN10987,Chen21e-B.1.1.28,Figure 1c,Virus isolate,IC50,7,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933 + REGN10987,Chen21e-B.1.429,Figure 1c,Virus isolate,IC50,4,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933 + REGN10987,Chen21e-B.1.617.1,Figure 1c,Virus isolate,IC50,4,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933 + REGN10987,Chen21e-B.1.526/S477N,Figure 1c,Virus isolate,IC50,2,1,10000,1,ng/ml,2022-01-08
+Chen21e,REGN10933 + REGN10987,Chen21e-B.1.526/E4B4K,Figure 1c,Virus isolate,IC50,5,1,10000,1,ng/ml,2022-01-08
+Chen21e,2B04,WA1+S:614G,Figure 1c,Virus isolate,IC50,1,1,10000,1,ng/ml,2022-01-08
+Chen21e,2B04,WA1+S:501Y+614G,Figure 1c,Virus isolate,IC50,1,1,10000,1,ng/ml,2022-01-08
+Chen21e,2B04,Chen21e-B.1.1.7,Figure 1c,Virus isolate,IC50,1,1,10000,1,ng/ml,2022-01-08
+Chen21e,2B04,Chen21e-B.1.351,Figure 1c,Virus isolate,IC50,>10000,1,10000,1,ng/ml,2022-01-08

```

Comment:
Please change all >10000 to 10000 since `potency` column should be strict number. The > symbol is implied by the `potency_upper_limit` value 10000, which means any number ≥ 10000 are not neutralizing the virus.

Reply:
I have pushed changes to my forked repository.